### ISSUE TITLE
Can I get assistance to get QuickComic.qlgenerator to work on Catalina?

### ISSUE THREAD

**Issue by oguntunde** [MaddTheSane/Simple-Comic#5]
After building Simple-Comic I added QuickComic.qlgenerator directly to /Users/"me"/Library/QuickLook & /Library/QuickLook/QuickComic.qlgenerator.

Simple-Comic works (albeit a little slowly) on my MacBook Pro running Catalina 10.15.2, but I get just the app icon as previews for all my .cbr and .cbz files

I ran qlmanage -p on a file to get feedback from the OS and received the following output:

```
rob@Agba-MacBook-Pro ~ % qlmanage -p /Volumes/Movies\ Music\ Media/\ •\ Comics/\ •\ Image/Afar\ \(2017\)\ .cbr 
Testing Quick Look preview with files:
	/Volumes/Movies Music Media/ • Comics/ • Image/Afar (2017) .cbr
2020-01-17 09:28:33.603 qlmanage[10550:546028] *** CFMessagePort: bootstrap_register(): failed 1100 (0x44c) 'Permission denied', port = 0x8e03, name = 'com.apple.coredrag'
See /usr/include/servers/bootstrap_defs.h for the error codes.
2020-01-17 09:28:33.761 qlmanage[10550:546028] *** CFMessagePort: bootstrap_register(): failed 1100 (0x44c) 'Permission denied', port = 0x1471f, name = 'com.apple.tsm.portname'
See /usr/include/servers/bootstrap_defs.h for the error codes.
rob@Agba-MacBook-Pro ~ % 
```

Do you have any time to advise me on what to do to get the QuickComic QuickLook to work?

**Comment by nickv2002** [comment]
@oguntunde

**Comment by nickv2002** [comment]
Do you have this same problem with the notarized version?
[Downloaded here](https://towatchlist.com/files/Simple%20Comic%202020-01-12%2012-05-43.zip)

**Comment by manuelgrabowski** [comment]
@nickv2002 I'm on Catalina and QuickLook works fine for me with the version you posted here.

**Comment by manuelgrabowski** [comment]
@oguntunde You might wanna check out #7, we found a probable cause for QuickLook not working there. :)

**Comment by oguntunde** [comment]
Thank you for all the help, guys. I downloaded the latest notarized version, deleted the copy of QuickComic.qlgenerator I had in the /Library/QuickLook directory. I copied over the QuickComic.qlgenerator version from ~/Library/QuickLook, launched SimpleComic , reset the qlmanager with qlmanage -r and voila! my previews are back! 

They were a bit slow so I deleted The Unarchiver and two other comic viewing apps: YAC Reader & Comic Viewer. This sped up the preview.

Once again, thank you for your advice and hard work!
